### ISSUE TITLE
_fnCallbackFire triggers events who bubble

### ISSUE THREAD

**Issue by heannig** [DataTables/DataTables#755]
I use DataTables v1.10.9 with Scroller v1.4.0.
My scenario is the following: My datatable uses the **serverside** option plus the scroller extension. The table also has nested tables which can be expanded dynamicly. The nested table is initialized and displayed, when you click on the expansion-button in a row. 
The problem is: during the initialization of the nested table, an **init.dt** event is fired on itself. That event bubbles to the parent table (that one with the scroller) which leads to an remeasurement and ajax-reload by the scroller extension (see code). In the end, the nested table is hidden by the parent table immediatly due to redrawing after the ajax-response

```
// dataTables.scroller.js
$(this.s.dt.nTable).on( 'init.dt', function () {
			that.fnMeasure();
		} );
```
Long story short: I think this is a bug. The **init.dt** event should not bubble up to the parent table. I would suggest to change ```.trigger()``` to ```.triggerHandler()```

```
// jquery.dataTables.js  --> _fnCallbackFire
if ( eventName !== null ) {
			var e = $.Event( eventName+'.dt' );
	
			$(settings.nTable).triggerHandler( e, args );
	
			ret.push( e.result );
		}
```

**Comment by DataTables** [comment]
Actually `init` is one of the events that I really want to retain bubbling for since it is really handy to be able to listen for `$(document).on( 'init.dt', ... );` to initialise plug-ins. Most of the others I will likely have not bubble in a future major update.

I've [committed a change to Scroller](https://github.com/DataTables/Scroller/commit/311c04a3745d43fc786eb414b07ec19b10966dd5) that should address this.

However, I would note that you cannot have child rows in a Scroller table - its virtual height calculations would be wrong if there were anything other than the host table's rows.
